### PR TITLE
JBPM-5573: Stunner - Integration of latest KIE Forms API & implementations

### PR DESCRIPTION
**Pull request by pefernan**
- fixed API changes on forms side
- fixed testing issues

These PR's should be merged together:
https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/pull/364
https://github.com/droolsjbpm/kie-wb-common/pull/640
https://github.com/droolsjbpm/jbpm-console-ng/pull/609
https://github.com/droolsjbpm/jbpm-designer/pull/533
https://github.com/droolsjbpm/appformer/pull/66

### CODE REVIEW

**Comment by pefernan** [comment]
@wmedvede can you take a look...

**Comment by wmedvede** [comment]
+1

File: appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/resources/org/kie/appformer/formmodeler/codegen/view/impl/html/templates/datebox.mv [hunk 1]
```diff
@@ -1,7 +1,7 @@
             <div id="@{field.name}_form_group" class="form-group">
                 <label class="control-label" for="@{field.name}">@{field.label}</label>
                 <div id="@{field.name}" placeholder="@{field.placeHolder}"
-                    @if{field.readonly}
+                    @if{field.readoOly}
```
Comment (mbarkley) [inline appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/resources/org/kie/appformer/formmodeler/codegen/view/impl/html/templates/datebox.mv]: I think this is a typo.

**Comment by mbarkley** [comment]
Found a typo. Otherwise +1.

File: appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/resources/org/kie/appformer/formmodeler/codegen/view/impl/html/templates/datebox.mv (hunk 1 above)
Comment (pefernan) [inline appformer-form-modeler/appformer-form-modeler-codegen/appformer-form-modeler-codegen-impl/src/main/resources/org/kie/appformer/formmodeler/codegen/view/impl/html/templates/datebox.mv]: @mbarkley +1 you're right thanks!

**Comment by pefernan** [comment]
jenkins retest this

**Comment by csadilek** [comment]
jenkins retest this

**Pull request merged**